### PR TITLE
Supports building ol8 images

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
This is going to be dependent on https://github.com/plus3it/amigen8/pull/55

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go vagrant go

**Comment by lorengordon** [comment]
go vagrant go

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
Oh just remembered that we need to modify the manifest collection so OL8 does not identify as RHEL8...

**Comment by lorengordon** [comment]
go codebuild go

**Review by ferricoxide — COMMENTED** [review]
No Rocky or Alma builders?

File: spel/scripts/amigen8-build.sh [hunk 1]
```diff
@@ -71,22 +71,49 @@ function err_exit {
...
-    DEFAULTREPOS=(
-        # RHUI 3 repo names as of EL8.5 GA
-        rhel-8-baseos-rhui-rpms
+# Setup per-builder values
+case $( rpm -qf /etc/os-release --qf '%{name}' ) in
+    centos-linux-release | centos-stream-release )
+        BUILDER=centos-8stream
+
+        DEFAULTREPOS=(
+        baseos
```
[the remark this answers is not in the document]
Comment (lorengordon) [reply spel/scripts/amigen8-build.sh]: fixed!

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go vagrant go

**Review by ferricoxide — APPROVED** [review]

**Pull request merged**